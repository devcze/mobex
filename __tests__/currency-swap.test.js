/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import CurrencySwap from '../components/currency-swap.js'

describe('CurrencySwap', () => {
  it('CurrencySwap render', () => {
    const component = renderer.create(<CurrencySwap/>);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
