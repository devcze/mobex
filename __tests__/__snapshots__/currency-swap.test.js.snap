// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrencySwap CurrencySwap render 1`] = `
<div
  className="currency-swap__Switcher-sc-1tmmjvk-0 iltxXE"
>
  ↑↓
</div>
`;
